pipelines/pipeline_katello_31_to_33_upgrade.yml

```diff
@@ -68,6 +68,18 @@
 - hosts: pipeline-katello-3.1-3.3-centos7
   become: true
   vars:
+    bats_forklift_dir: "/home/vagrant/sync"
```

Comment:
I would too but lately this is the directory my stuff has been syncing too. I havent looked to see what is causing that though. Will revert for the tim ebeing